src/Core/Components/CounterBadge/FluentCounterBadge.razor.cs

```diff
@@ -35,8 +36,8 @@ public partial class FluentCounterBadge : FluentComponentBase, IDisposable
     /// Gets or sets the number displayed inside the badge.
     /// Can be enriched with a plus sign with <see cref="ShowOverflow"/>
     /// </summary>
-    [Parameter, EditorRequired]
-    public int? Count { get; set; }
+    [Parameter]
+    public int Count { get; set; } = 0;
```

Comment:
Why to remove the `nullable` value? To avoid breaking-change, is it possible to test this nullable value?

Reply:
Changed that back